src/ln/channel.rs

```diff
@@ -1114,7 +1117,9 @@ impl Channel {
 		if msg.max_accepted_htlcs < 1 {
 			return Err(HandleError{err: "0 max_accpted_htlcs makes for a useless channel", action: None});
 		}
-
+		if msg.max_accepted_htlcs > 483 {
+			return Err(HandleError{err: "max_accpted_htlcs > 483", action: None});
```

Comment:
We need to take some action here, probably Disconnect, I guess.

Reply:
In Bolt, for `open_channel`, it uses the term 'fail' (the channel), but 'reject' (the channel) for  `accept_channel`. Is there any difference beside Disconnect'ing?

Reply:
Probably not. "reject channel" only appears in "accept_channel message" section.
I will fill out ErrorAction.

Reply:
Yea, I think its the same, I kinda dont like the general "disconnect on channel-open-error" model but that seems to be the only way to handle things pre-funding.

Reply:
Maybe we should do a distinction between disconnecting the peer and aborting the channel ? Seems to that we don't want to disconnect the peer if we already share sane channels with it. Sending an ErrorMessage could be better in this case

Reply:
Oops, indeed, I think I hadn't saw the section on error message handling that says that you must fail the channel. In that case it probably makes sense to just send an error message and drop the channel instead of disconnecting.